tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -134,26 +134,41 @@ - (void)addParentTo:(NSProgress*)child {
     ASSERT_EQ(YES, [progress isIndeterminate]);
 
     [progress setTotalUnitCount:5];
-    ASSERT_EQ(YES, [progress isIndeterminate]);
+    ASSERT_EQ(NO, [progress isIndeterminate]);
+
+    [progress setCompletedUnitCount:5];
+    ASSERT_EQ(NO, [progress isIndeterminate]);
+
+    [progress setTotalUnitCount:0];
+    ASSERT_EQ(NO, [progress isIndeterminate]);
 
```

Comment:
(No reason to ASSERT unless it must stop the test.)

---
In reply to: [73764722](https://github.com/Microsoft/WinObjC/pull/836#discussion_r73764722) [](ancestors = 73764722)